syft/frameworks/torch/tensors/abstract.py

```diff
@@ -14,6 +15,35 @@ def wrap(self):
         wrapper.is_wrapper = True
         return wrapper
 
+    def serialize(
+        self, compress=True, compress_scheme=0
```

Comment:
Put these on separate lines or the entire def on 1 line

Reply:
fixed